lib/datasource/hex/index.js

```diff
@@ -12,15 +12,17 @@ function getHostOpts() {
 }
 
 async function getPkgReleases({ lookupName }) {
-  const hexUrl = `https://hex.pm/api/packages/${lookupName}`;
+  let depName = '';
+  if (lookupName) depName = lookupName.split(':')[0];
+  const hexUrl = `https://hex.pm/api/packages/${depName}`;
```

Comment:
This logic doesn't really make sense. If there's no lookupName then you'll just query an empty string?